.gitlab-ci.yml

```diff
@@ -1,3 +1,14 @@
+stages:
+  - stage0
+  - stage1
+  - stage2
+  - stage3
+  - stage4
+  - stage5
+  - stage6
+  - stage7
+  - stage8
```

Comment:
I am worried that the pipeline is too deep and CI builds take too long. But we will see.